src/WinRT.Runtime/ApiCompatBaseline.net8.0.txt

```diff
@@ -1,4 +1,6 @@
 Compat issues with assembly WinRT.Runtime:
 CannotChangeAttribute : Attribute 'System.AttributeUsageAttribute' on 'WinRT.GeneratedBindableCustomPropertyAttribute' changed from '[AttributeUsageAttribute(AttributeTargets.Class, Inherited=false, AllowMultiple=false)]' in the contract to '[AttributeUsageAttribute(AttributeTargets.Class | AttributeTargets.Struct, Inherited=false, AllowMultiple=false)]' in the implementation.
 CannotRemoveAttribute : Attribute 'System.ComponentModel.EditorBrowsableAttribute' exists on 'WinRT.WinRTExposedTypeAttribute' in the contract but not the implementation.
-Total Issues: 2
+ParameterNamesCannotChange : Parameter name on member 'WinRT.ActivationFactory.Get(System.String)' is 'activatableClassId' in the implementation but 'typeName' in the contract.

```

Comment:
Are these parameter changes really needed if they are technically API compat concerns?

Reply:
Not strictly needed, but it was more for consistency. These would really only be breaking if someone is recompiling from source and happened to have an invocation with a named argument. Otherwise there's no concerns with binary compatibility. I can revert if you want, but I was thinking it should be fine? Let me know!

Reply:
My thinking is given the old parameter name wasn't incorrect either, I wouldn't add an API compat change just for this.

Reply:
Reverted! 👍